### PR TITLE
custom error pages

### LINKED ISSUE #78: Custom error pages
It would be nicer to show a custom error page rather than default for 500, 404 , 401 errors eg. `catalog/invalid_id`

### PR DESCRIPTION
**Pull request by banukutlu**
Closes #78

### CODE REVIEW

**Review by whereismyjetpack — DISMISSED** [review]

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class ApplicationController < ActionController::Base
   include Blacklight::Controller
   layout :determine_layout if respond_to? :layout
 
-  rescue_from CanCan::AccessDenied, with: :render401
+  rescue_from CanCan::AccessDenied, with: :unauthorized
```
Comment (ajkiessl) [inline app/controllers/application_controller.rb:8]: I just took a quick glance at this so I might not be correct about this, but I think the error is coming from this ☝️ .  FilesController is inheriting from ApplicationController and there is no unauthorized method there.  I think this should route to the ErrorsController in some way.

Comment (banukutlu) [reply app/controllers/application_controller.rb]: yup right, I missed that, thanks @ajkiessl @whereismyjetpack! I just added a fix.

**Comment by banukutlu** [comment]
CC not happy, will add specs for errors_controller

**Comment by whereismyjetpack** [comment]
I'm ok merging  `-0.1%` change isn't a terribly good threshold for blocking a merge. we should perhaps tweak that

**Comment by banukutlu** [comment]
> I'm ok merging `-0.1%` change isn't a terribly good threshold for blocking a merge. we should perhaps tweak that

yeah agreed, also not sure what it is showing as negative change, it looks like all good https://codeclimate.com/github/psu-libraries/etda_explore/compare/86db3adc5b8af0419c5a0a3dd18d7f23a31f7f54...2445b61d6e3415eafd85b557cfb491f70bf29e38#ratings

**Comment by banukutlu** [comment]
> I'm ok merging `-0.1%` change isn't a terribly good threshold for blocking a merge. we should perhaps tweak that

yeah agreed, also not sure what it is showing as negative change, it looks like all good https://codeclimate.com/github/psu-libraries/etda_explore/compare/86db3adc5b8af0419c5a0a3dd18d7f23a31f7f54...2445b61d6e3415eafd85b557cfb491f70bf29e38#ratings

**Pull request merged**